templates/index.html

```diff
@@ -454,34 +454,116 @@ <h4 class="modal-title">העדפות:</h4>
+                            <label style="vertical-align: middle; height: 30px; display: table-cell;" for="prefRadius">רדיוס:</label>                                          
+                        </div>   
+                        <div style="float: right; margin-right: 10px">
+                            <select id="prefRadius" style="width: 80px; vertical-align: middle; display: table-cell; width: 100px;">
+                                <option id="prefRadius500" value="500">500</option>
+                                <option id="prefRadius1000" value="100">1000</option>
+                                <option id="prefRadius1500" value="1500">1500</option>
+                                <option id="prefRadius2000" value="2000">2000</option>
+                            </select>
+                        </div>  
+                     </div>                                    
+                </div>     
+                
+                <div style="width: 400px; margin: 0 auto;" class="prefReportControls">    
+                     <div style="margin-right: 15px">                                
+                        <div style="clear: right; float: right;  margin-left: 10px; width: 100px">                     
+                            <label style="vertical-align: middle; height: 30px; display: table-cell;" for="prefAccidentSeverityForReport">חומרת תאונה</label>                                          
+                        </div>   
+                        <div style="float: right; margin-right: 10px">
+                            <select id="prefAccidentSeverityForReport" style="width: 80px; vertical-align: middle; display: table-cell; width: 100px;">
+                                <option id="prefReportLight" value="2">קלה</option>
+                                <option id="prefReportSevere" value="1">חמורה</option>
+                                <option id="prefReportFatal" value="0">קטלנית</option>                          
+                            </select>
+                        </div>  
+                     </div>                                    
+                </div>      
+                         
+                <div style="width: 400px; margin: 0 auto;" class="prefReportControls">    
+                     <div style="margin-right: 15px">                                
+                        <div style="clear: right; float: right;  margin-left: 10px; width: 100px">                     
+                            <label style="vertical-align: middle; height: 30px; display: table-cell;" for="prefHistoricalReportPeriod">דוח היסטורי עבור</label>                                          
+                        </div>   
+                        <div style="float: right; margin-right: 10px">
+                            <select id="prefHistoricalReportPeriod" style="width: 100px; vertical-align: middle; display: table-cell;">
```

Comment:
I fixed the rest of the issues.
How can I use the jijna2 template? This screen is a bootstrap modal screen. 
Do you mean getting the template through ajax, as the rest of the data and rendering it?